packages/core/src/Stores/common-store.js

```diff
@@ -255,6 +264,37 @@ export default class CommonStore extends BaseStore {
         this.app_routing_history.unshift(router_action);
     }
 
+    changeLanguage = (key, changeCurrentLanguage) => {
```

Comment:
changeLanguage function also exists in core/src/Utils/Language/language.js. Can we remove one of it?

Reply:
Will do